fair/run.py

```diff
@@ -164,6 +168,11 @@ def run_command(
     _run_executable = "script" in _cfg["run_metadata"] or "script_path" in _cfg["run_metadata"]
     _run_executable = _run_executable and mode == CMD_MODE.RUN
 
+    if mode == CMD_MODE.PASS:
+        _logger.debug(
+            "Run called in passive mode, no command will be executed"
```

Comment:
It's not fully passive, so maybe just call it CI mode?
```suggestion
            "Run called in CI mode, no command will be executed"
```

Reply:
Considering the sole purpose of `run` is to run a model execution/set of executions, I would argue that this is indeed passive. The reason I did not state "CI" is that at this level, if this script of code is isolated what is happening is the actual "run bash command" part of `run_command` is being skipped